website/docs/docs/core-versions.md

```diff
@@ -39,26 +42,19 @@ After a minor version reaches the end of its critical support period, one year a
 
 We aim to release a new minor "feature" every 3 months. _This is an indicative timeline ONLY._ For the latest information about upcoming releases, including their planned release dates and which features and fixes might be included in each, always consult the [`dbt-core` repository milestones](https://github.com/dbt-labs/dbt-core/milestones).
 
-| dbt Core   | Initial Release | Latest Patch         | Active Support Until | Critical Support Until  |
-|------------|-----------------|----------------------|----------------------|-----------------|
-| 1.0.x      | 2021-12-03      | 1.0.1                | 1.1.0 release        | 2022-12-03      |
-| _1.1.x_    | _2022-04_       |                      | _1.2.0 release_      | _2023-04_       |
-| _1.2.x_    | _2022-07_       |                      | _1.3.0 release_      | _2023-07_       |
-| _1.3.x_    | _2022-10_       |                      | _1.4.0 release_      | _2023-10_       |
-
-_Italics: Future releases, NOT definite commitments. Shown for indication only._
+<Snippet src="core-versions-table" />
 
 ## Best practices for upgrading
 
-Because of our new version practice, we've outline best practices and expectations for dbt users to upgrade as we continue to release new versions of dbt Core.
+Because of our new version practice, we've outlined best practices and expectations for dbt users to upgrade as we continue to release new versions of dbt Core.
 
 ### Upgrading to new patch versions
 
-We expect users to upgrade to patches as soon as they're available. When we refer to a "minor version" of dbt Core, which may be written as v1.0 or v1.0.x, we are always referring to _the latest patch release of that minor version_. We highly encourage you to structure your development and production environments such that you can always install the latest patches of `dbt-core` and any adapter plugins (noting that those patch numbers may be different).
+We expect users to upgrade to patches as soon as they're available. When we refer to a "minor version" of dbt Core, such as v1.0, we are always referring to the latest available patch release for that minor version. We encourage you to structure your development and production environments so that you can always install the latest patches of `dbt-core` and any adapter plugins. (Note that patch numbers may be different between dbt-core and plugins. [See below](#how-we-version-adapter-plugins) for an explanation.)
```

Comment:
Should we add something here about patch releases only being small, targeted bug and security fixes so that it seems less scary to auto-grade to them?

Reply:
Good call!